src/percentages.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @desc Adds a blank new column to the markbook for displaying percentages
+ */
+const addPercentageColumn = () => {
+    // add percent header
+    $('<td class="mwTABLE_CELL_HEADER tdAchievement" style="font-weight: bold" align="center">Percent</td>').insertAfter('#markbookTable td:first');
+    
+    // narrow assignment name column width
+    $('#markbookTable tr:not(:first)').each(function () {
+        $(this).find('td:first').css('width', '250px');
+    });
+
+    // copy date column (to keep row style) and paste it after the first column
+    $('#markbookTable tr:not(:first)').each(function () {
+        const dateCell = $(this).find('td:nth-last-child(3)');
+        $(dateCell).clone().insertAfter($(this).find('td:first'));
+    });
+
+    // clear the column of any text and change the font to the same as input boxes
+    $('#markbookTable tr:not(:first) > td:nth-child(2)').each(function () {
+        $(this).text('');
+        $(this).css({
+            'font-family': 'Monaco, Courier, monospace'
+        });
+    });
+};
```

Comment:
![image](https://user-images.githubusercontent.com/43765559/102732316-99d28c80-42f7-11eb-8fba-2f7630f2ba53.png)
Move the percentage column to the rightmost side so the user is not confused.

Reply:
```suggestion
const addPercentageColumn = () => {
    // add percent header
    $('<td class="mwTABLE_CELL_HEADER tdAchievement" style="font-weight: bold" align="center">Percent</td>').insertAfter('#markbookTable tr:first td:last');
    
    // narrow assignment name column width
    $('#markbookTable tr:not(:first)').each(function () {
        $(this).find('td:first').css('width', '250px');
    });

    // copy end column (to keep row style) and paste it after the last column
    $('#markbookTable tr:not(:first)').each(function () {
        const cell = $(this).find('td:nth-last-child(1)');
        $(cell).clone().insertAfter($(this).find('td:last'));
    });

    // clear the column of any text and change the font to the same as input boxes
    $('#markbookTable tr:not(:first) > td:nth-child(6)').each(function () {
        $(this).text('');
        $(this).css({
            'font-family': 'Monaco, Courier, monospace'
        });
    });
};
```
This is how I implemented for testing, feel free to change the jquery selectors for more readability